### PR TITLE
fix(DTFS2-6520): made aria-labels in Deal page tabs headings more consistent

### PR DESCRIPTION
**Pull request by avaitonis**
### Introduction
Visible H2 text didn't match aria-label in Deal underwriting tab.

### Resolution
 - Removed `aria-label` for all Deal page tabs. 
 - Changed Underwriting tab header text from `Deal` to `Underwriting`. Agreed with Tim.
 - Markup cleanup

### CODE REVIEW

File: trade-finance-manager-ui/templates/case/activity/activity.njk [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
     <div class="govuk-grid-column-full">
 
-      <h2 class="govuk-heading-xl govuk-!-margin-top-6 govuk-!-margin-bottom-5" data-cy="activity-heading" aria-label="Activity and comments">Activity and comments</h2>
+      <h2 class="govuk-heading-xl govuk-!-margin-top-6 govuk-!-margin-bottom-5" data-cy="activity-heading" aria-label="Deal activity and comments">Activity and comments</h2>
```
Comment (lmarrai-sw) [inline trade-finance-manager-ui/templates/case/activity/activity.njk:20]: For all of these changes, @avaitonis we could just remove the `aria-label` and the actual text will be read out instead.

Do you think the actual text isn't clear enough and the text in the `aria-label` is better? If so then we probably want to change the actual text too (and can talk to the DTFS service designer about that)

**Review by lmarrai-sw — CHANGES_REQUESTED** [review]

File: trade-finance-manager-ui/templates/case/activity/activity.njk (hunk 1 above)
Comment (avaitonis) [reply trade-finance-manager-ui/templates/case/activity/activity.njk]: Hi @lmarrai-sw ,

aria-labels are removed, text copy change agreed with Tim.

**Review by lmarrai-sw — APPROVED** [review]
Thanks for this @avaitonis

**Review by lmarrai-sw — APPROVED** [review]
Thanks @avaitonis

**Pull request merged**